src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -252,11 +252,6 @@ public enum NuGetLogCode
         /// </summary>
         NU1503 = 1503,
 
-        /// <summary>
-        /// A VersionOverride was specified for a PackageReference but central package management is not enabled so the value was ignored.
```

Comment:
Do we really need to remove this NU1504? Probably just keep it until next solution, then reword that time. 
Also, it could be confusing end consumers without this patch who is still getting NU1504 warning, since it doesn't exist anywhere.

Reply:
I don't think we'll ever bring it back since there's no good way for us to distinguish between intentional and unintentional usage of the feature unless we rename the metadata or add a new property to enable the warning.  At the point we can bring the warning code back.  It was never released or documented and is only showing up for people running the internal preview.